src/ProfilesList.json

```diff
@@ -399,5 +399,6 @@
   "Ken-Musau.json",
   "agtorres3.json",
   "Meko007.json",
-  "brianobot.json"
+  "brianobot.json",
+  "NanditaSunil.json"
```

Comment:
```suggestion
  "Nanditasunil.json"
```

@Nanditasunil Match it the filename you created

Reply:
Done , Please Check